tools/releaseBuild/Images/GenericLinuxFiles/PowerShellPackage.ps1

```diff
@@ -77,6 +79,22 @@ function BuildPackages {
 
         if ($TarX64) { Start-PSPackage -Type tar @releaseTagParam -LTS:$LTS }
 
+        if ($TarMinSize) {
+            Write-Host "---- Min-Size ----"
```

Comment:
Any reason to use `Write-Host` instead of `Write-Verbose -Verbose` like other places in the script.

Reply:
No particular reason. I can change to `Write-Verbose` if that's preferred.